flowlayout/src/main/java/com/google/accompanist/flowlayout/Flow.kt

```diff
@@ -82,6 +93,17 @@ public fun FlowRow(
  * @param lastLineMainAxisAlignment Overrides the main axis alignment of the last column.
  */
 @Composable
+@Deprecated(
+    """
+accompanist/FlowColumn is deprecated.
+For more migration information, please visit https://google.github.io/accompanist/flowlayouts/#migration
+""",
+    replaceWith = ReplaceWith(
+        "FlowColumn",
```

Comment:
Does this work? I tried a similar thing with the Pager component, but it doesn't seem to work nicely with optional arguments. Wondering if I'm missing something...

Reply:
It works fine for me without any compiler errors. You can definitely share a code where you're having an issue with optional arguments.